src/corehost/cli/runtime_config.h

```diff
@@ -11,6 +11,13 @@
 
 typedef web::json::value json_value;
 
+enum roll_fwd_on_no_candidate_fx_option
+{
+    roll_fwd_disabled = 0,
+    roll_fwd_minor,
+    roll_fwd_major_or_minor
+};
+
```

Comment:
Consider `enum class` (scoped enum) and no `roll_fwd_` prefixes on members.

Reply:
Done. Thanke